docs/md/staged-workload.md

```diff
@@ -1,29 +1,32 @@
 # Staged Workloads
 
-A staged workload is a discrete body of work, which takes data from a source,
+A staged workload is a continuous body of work which takes data from a source,
```

Comment:
OK. (Not sure what to make of this.)

Reply:
I suspect the "discrete" was originally copypasta'd from this documentation on WGS:

https://github.com/broadinstitute/wfl/blob/develop/docs/md/modules-general.md?plain=1#L17-L20

Most of the time (with `TerraDataRepoSource`) a staged workload is processing data continuously, and we do not have prior knowledge of the scope of its processing.  But a staged workload could have a `TDRSnapshotListSource`, its processing would be discrete in this case.

TL;DR I'll remove this descriptor.